packages/management/addons-manager/bundle/config/upstream/addons-manager.yaml

```diff
@@ -20,6 +20,17 @@ rules:
   - get
```

Comment:
Question: Are we going to use this same SA for all the config controllers and CB controller?

Reply:
I think has to be the same service account since all controllers running in the same binary.